Specs/DateTime/English/DatePeriodParser.json

```diff
@@ -5121,4 +5121,179 @@
     }
   ]
 }
+,
+{
+  "Input": "Please find us a time to meet earlier in the week",
```

Comment:
Can you change this for a case where there's at least two days in the range?
Also, the sentence doesn't make sense. Maybe something like "we could have scheduled a time to meet earlier in the week"